pvlib/test/test_solarposition.py

```diff
@@ -499,3 +498,58 @@ def test_analytical_azimuth():
     azimuths = solarposition.solar_azimuth_analytical(*test_angles.T, zenith=zeniths)
 
     assert not np.isnan(azimuths).any()
+
+
+@pytest.fixture()
+def expected_rise_set_geometric():
+    # for Golden, CO, from USNO websites
```

Comment:
Update source for solution, this isn't from USNO since that site only displays minutes